tools/RAiDER/utilFcns.py

```diff
@@ -520,36 +517,36 @@ def writePnts2HDF5(lats, lons, hgts, los, lengths, outName='testx.h5', chunkSize
         f['Rays_len'].attrs['MaxLen'] = np.nanmax(lengths)
 
 
-def writeWeatherVars2HDF5(lat, lon, x, y, z, q, p, t, proj, outName=None):
```

Comment:
I don't think this function is used or needed anymore, so if nothing is calling it it can be deleted.

Reply:
I had a feeling. I didn't see anything in the project calling it, but because it was such a big function I was reluctant to remove it entirely in case it was still being worked on. If it's just not used anymore, I'll delete it.